src/canvas/__tests__/__snapshots__/eventData.test.ts.snap

```diff
@@ -145,17 +145,17 @@ exports[`Canvas event data > HTML event "dragend" should fire a corresponding ca
   [
     "after:render",
     {
-      "ctx": CanvasRenderingContext2D {
+      "ctx": {
```

Comment:
this is the biggest change
snapshotting ctx is not same in browser and jsdom
in jsdom properties are on instance
in browser they are on prototype

so I had to write a custom serializer for the canvas rendering context so it is stable cross env between jsdom and browser

otherwise in browser it is snapshotted as `{}` which fails the snapshot

Reply:
Maybe the custom serializer can return just the string with the type? we don't need to be accurate here. after:render pass the ctx where the rendering event happened. We don't need to know that ctx has determinated properties

Reply:
yeah I guess that would be the easiest
what exactly would you want to output here?
what is relevant for the test?
so I can adjust the serializer